apps/server/package.json

```diff
@@ -18,6 +18,7 @@
     "multer": "^1.4.5-lts.1",
     "node-osc": "^9.1.4",
     "ontime-utils": "workspace:*",
+    "osc-min": "^2.1.1",
```

Comment:
we probably dont want 2 dependencies for OSC
if we migrate the server as well I suggest we just merge this and release with the automations